File: sakila-payment-service/src/main/java/com/example/sakila/module/customer/event/CustomerEventUtils.java
package com.example.sakila.module.customer.event;

import com.example.sakila.module.customer.Customer;
import com.example.sakila.module.customer.event.model.CustomerEventDTO;

public class CustomerEventUtils {

  public static CustomerEventDTO toDTO(Customer customer) {
    CustomerEventDTO customerDTO = new CustomerEventDTO();

    customerDTO.setId(customer.getId());
    customerDTO.setStoreId(customer.getStoreId());
    customerDTO.setFirstName(customer.getFirstName());
    customerDTO.setLastName(customer.getLastName());
    customerDTO.setEmail(customer.getEmail());
    customerDTO.setAddressId(customer.getAddressId());
    customerDTO.setActiveBool(customer.getActiveBool());
    customerDTO.setCreateDate(customer.getCreateDate());
    customerDTO.setLastUpdate(customer.getLastUpdate());
    customerDTO.setActive(customer.getActive());

    return customerDTO;
  }
}

File: sakila-payment-service/src/main/java/com/example/sakila/module/customer/event/CustomerEventUtils.kt
package com.example.sakila.module.customer.event

import com.example.sakila.module.customer.Customer
import com.example.sakila.module.customer.event.model.CustomerEventDTO

class CustomerEventUtils {

  companion object {

      @JvmStatic
      fun toDTO(customer: Customer): CustomerEventDTO {
        val customerEventDTO = CustomerEventDTO()

        customerEventDTO.id = customer.id
        customerEventDTO.storeId = customer.storeId
        customerEventDTO.firstName = customer.firstName
        customerEventDTO.lastName = customer.lastName
        customerEventDTO.email = customer.email
        customerEventDTO.addressId = customer.addressId
        customerEventDTO.activeBool = customer.activeBool
        customerEventDTO.createDate = customer.createDate
        customerEventDTO.lastUpdate = customer.lastUpdate
        customerEventDTO.active = customer.active

        return customerEventDTO
      }
  }

}
